web/docs/guides/database/tables.mdx

```diff
@@ -348,8 +336,165 @@ create table salaries (
 );
 ```
 
+## Views
```

Comment:
I think we need a mention of VIEWS + RLS, a:

```sql
ALTER VIEW your_view OWNER TO authenticated;
```

Is needed for RLS to work ([ref](https://github.com/supabase/supabase/discussions/901#discussioncomment-500895)). That will change in PG 15.

Reply:
ah great, thanks Steve! I just copied this from an old blog. I should actually add RLS to this whole page

Reply:
@steve-chavez - I just added the `caution` below. I didn't actually realise you could enable RLS by changing the owner to "authenticated". Thanks for the tip!

Reply:
@kiwicopple Added a new comment below, if you agree with the change then this should be good to merge.